components/table/Status.vue

```diff
@@ -19,4 +19,9 @@ const fieldStatus = [
   { name: "Aan het finishen", class: "text-yellow-500 animate-pulse" },
   { name: "Geannuleerd", class: "text-red-500" },
 ];
+
+function formatTime(dateTimeStr: string | null) {
+  if (!dateTimeStr) return "";
```

Comment:
```suggestion
  if (!dateTimeStr) return fieldStatus[0].name;
```
Dan zien ze iig de status (als er geen tijd is)